src/base/operations/cancel_change_role_request_builder.js

```diff
@@ -0,0 +1,29 @@
+import { default as xdr } from '../generated/xdr_generated'
+import { BaseOperation } from './base_operation'
+import { UnsignedHyper } from 'js-xdr'
+
+export class CancelChangeRoleRequestBuilder {
+  /**
+   * Creates operation to cancel change role request
+   * @param {object} opts
+   * @param {string} opts.requestID - ID of the request
+   * @param {string} [opts.source] - The source account for the operation.
+   * Defaults to the transaction's source account.
+   * @returns {xdr.CancelChangeRoleRequestOp}
+   */
+  static cancelChangeRoleRequest (opts) {
+    let cancelChangeRoleRequestOp = new xdr.CancelChangeRoleRequestOp({
```

Comment:
pls use validators `src/utils/validators.js` to validate input parameters. It is very important cuz, `UnsignedHyper.fromString` for example, throws an absolutely meaningless error if provide it with any non-string parameter.